test/integration/conftest.py

```diff
@@ -149,6 +158,11 @@ def db_scripts() -> list[ExaFunction]:
         ExaFunction(
             name="fibonacci",
             comment="emits Fibonacci sequence of the given length",
+            inputs=[ExaParameter(name="seq_length", type="DECIMAL(18,0)")],
+            emits=[
+                ExaParameter(name="NUM", type="DECIMAL(18,0)"),
```

Comment:
do you normalize the types somewhere, because the body defines INTEGER and here you define DECIMAL, or do I miss something

Reply:
INTEGER becomes decimal when I read it back